docs/_data/componentLibraries.js

```diff
@@ -101,6 +101,12 @@ const componentLibraries = [
     description:
       'Lion web components is a set of highly performant, accessible and flexible Web Components. They provide an unopinionated, white label layer that can be extended to your own layer of components.',
   },
+  {
+    name: 'LRNWebcomponents',
+    url: 'https://github.com/elmsln/lrnwebcomponents',
```

Comment:
Do you have a public URL that more directly points to the components in the library, and possibly how to consume them?

Reply:
https://webcomponents.psu.edu/ isn't comprehensive but has a lot of them, you mean like that?

Reply:
That definitely gave devs a bette rides of what they might be able to get from the library, yes